en/docs/reference/mi-security-reference/using_keystores.md

```diff
@@ -1,4 +1,4 @@
-# Using Keystores
+    # Using Keystores
```

Comment:
We don't need the space here

Reply:
fixed [d7baf75](https://github.com/wso2/docs-apim/pull/7325/commits/d7baf7539c3a8b3574cfc6f35b50f9285f28b3a2)